### ISSUE TITLE
macOS (Darwin) arm64 binary variant?

### ISSUE THREAD

**Issue by pirouet** [oracle/node-oracledb#1687]
1. What versions are you using? 6.5.1

2. Describe the problem
With the introduction of the arm64 macOS Instant Client, is there a timeline to have build of oracledb that is compatible in Thick mode? It is not always possible to use thin mode as much as I would like to do so for legacy application that cannot support the latest encryption settings/mTLS. 

Running with latest oracledb version, this is the output in console:

```
Error: NJS-045: cannot load a node-oracledb Thick mode binary for Node.js. Please try using Thin mode. 
  Looked for {path}/node_modules/oracledb/build/Release/oracledb-6.5.1-darwin-arm64.node, {path}/node_modules/oracledb/build/Release/oracledb.node, {path}/node_modules/oracledb/build/Debug/oracledb.node, {path}/node_modules/oracledb/lib/node_modules/oracledb/build/Release/oracledb-6.5.1-darwin-arm64.node, {path}/node_modules/oracledb/lib/node_modules/oracledb/build/Release/oracledb.node

```

**Comment by cjbj** [comment]
"Soon".

**Comment by sharadraju** [comment]
@pirouet macOS ARM binary is available with node-oracledb 6.6
